clients/python-wrapper/pyproject.toml

```
@@ -44,7 +44,7 @@ test = [
 Homepage = "https://github.com/treeverse/lakeFS/tree/master/clients/python-wrapper"
 
 [tool.setuptools.packages.find]
-include = ["lakectl", "lakefs"]
+include = ["lakectl*", "lakefs*"]
```

Just have exact match of the packages
```suggestion
include = ["lakectl", "lakectl.*", "lakefs", "lakefs.*"]
```